ql/src/java/org/apache/hadoop/hive/ql/Driver.java

```diff
@@ -558,11 +560,7 @@ private void setTriggerContext(String queryId) {
     // query info is created by SQLOperation which will have start time of the operation. When JDBC Statement is not
     // used queryInfo will be null, in which case we take creation of Driver instance as query start time (which is also
     // the time when query display object is created)
-    if (driverContext.getQueryInfo() != null) {
-      queryStartTime = driverContext.getQueryInfo().getBeginTime();
-    } else {
-      queryStartTime = driverContext.getQueryDisplay().getQueryStartTime();
-    }
+    queryStartTime = driverContext.getQueryStartTime();
```

Comment:
not sure why we need this change, but now above comment doesn't make any sence

Reply:
just a refactor, to move this logic to a separate method, but yeah, the original comment has lost its meaning here, let me move it also and rename if needed